src/pudl/transform/nrelatb.py

```diff
@@ -0,0 +1,368 @@
+        "rate_of_return_on_equity_nominal": "rate_of_return_on_equity_nominal",
+        "additional_occ": "capex_overnight_additional",
+        "wacc_nominal": "wacc_nominal",
+        "wacc_real": "wacc_real",
+        "cfc": "construction_finance_factor",
+        "lcoe": "levelized_cost_of_energy",
+        "heat_rate_penalty": "heat_rate_penalty",
+    }
+    nrelatb = (
+        raw_nrelatb__data.replace(["*", ""], pd.NA)
+        .pipe(
+            helpers.fix_boolean_columns,
+            boolean_columns_to_fix=list(raw_nrelatb__data.filter(regex=r"^is_")),
+        )
+        ## clean & rename values in core_metric_parameters
+        .pipe(helpers.simplify_strings, ["core_metric_parameter"])
+        .pipe(helpers.cleanstrings_snake, cols=["core_metric_parameter"])
+        .replace({"core_metric_parameter": core_metric_parameters_rename})
+        # we are droping dropping records that are completely dupes - not just
+        # dupes based on IDX_ALL
+        .drop_duplicates(keep="first")
+    )
+    assert not any(nrelatb.duplicated(IDX_ALL))
+
+    # ensure that we have the same set of parameters in the unstackers and in the rename
+    params_found = set(nrelatb.core_metric_parameter.unique())
+    params_rename = set(core_metric_parameters_rename.values())
+    params_unstacker = set(Unstacker().params_all)
+    for params in [params_rename, params_unstacker]:
+        if param_differences := params_found.symmetric_difference(params):
+            raise AssertionError(
+                "Found different parameters in the NREL ``core_metric_parameter`` column "
+                "compared to the params in the TableUnstackers or core_metric_parameters_rename:\n"
+                f"{param_differences=}"
+            )
+    return nrelatb.convert_dtypes()
+
+
+@asset
+def _core_nrelatb__yearly_projections_rates(
+    _core_nrelatb__transform_start,
+) -> pd.DataFrame:
+    """Transform the yearly rates table.
+
+    Right now, this just unstacks the table.
+    """
+    df = transform_unstack(_core_nrelatb__transform_start, Unstacker().rate_table)
+    return df
+
+
+@asset
+def _core_nrelatb__yearly_projections_by_scenario(
+    _core_nrelatb__transform_start,
+) -> pd.DataFrame:
+    """Transform the yearly NREL ATB scenario table.
+
+    Right now, this just unstacks the table.
+    """
+    df = transform_unstack(_core_nrelatb__transform_start, Unstacker().scenario_table)
+    return df
+
+
+@asset
+def _core_nrelatb__yearly_projections_by_technology_detail(
+    _core_nrelatb__transform_start,
+) -> pd.DataFrame:
+    """Transform the yearly technology detail table.
+
+    Right now, this just unstacks the table.
+    """
+    df = transform_unstack(
+        _core_nrelatb__transform_start, Unstacker().tech_detail_table
+    )
+    return df
+
+
+@asset
+def _core_nrelatb__yearly_revisions(
+    _core_nrelatb__transform_start: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform small table with the r."""
```

Comment:
Another flag for "don't forget to finish this thought!"

Reply:
lol yes ty!!